pages/design/review.md

```diff
@@ -1,5 +1,21 @@
 ---
-title: Auto-WCAG Rule Review Process
+title: Auto-WCAG Rule Design and Review Process
 ---
 
-TODO
\ No newline at end of file
+## Step 1: Creating the proposal in an Issue
+
+Every rule is designed by a team of two or three people. They do their work in a github issue. This means that every activity is represented by an open issue on Github. The first post of an issue contains the proposal they are working on. In subsequent posts, discussions are held, and as they are resolved, the draft at the top of the issue is updated. Once the group is done with the proposal, a pull request is created, and the issue is closed.
+
+## Step 2: Propose a change in a Pull Request
+
+When a proposal in step 1 is done, a pull request is created to update the rule in `_drafts`. This gives the rest of auto-WCAG the opportunity to respond to the proposal. At this point one of three things can happen:
+
+1. Editorial comments can be resolved immediately by updating the pull request. 
+
+2. For any comments that can't be resolved immediately, a new issue is created. Once the review period of the pull request has passed, the pull request is merged in. The team working on this rule continues working on the new issue that was created, going back to step 1.
+
+3. If no comments come in, the issue can be merged in after the review period. If the authors feel the rule is read for publication, after they merged the PR into draft, they can open a new PR, which must have "FINAL" in the PR name, to move the rule from `_drafts` to `_rules`. This will publish the rule. The rule needs at least **3 approved votes** before it can be merged from people who weren't the editors. If new comments come in, the PR must be closed without merging, and a new issue is created to work on the fix.
```

Comment:
"read" -> "ready"

Reply:
Would change "after they merged the PR into draft" to "and they have merged the PR into draft", as I think that will make the parenthetical sentence slightly more easy to read and understand right away.

Also, up until this paragraph we've used "Pull Request" and not "PR". Should we stick with that?

Reply:
3. If no comments come in, the issue can be merged in after the review period. If the authors feel the rule is ready for publication, and they have merged the pull request into draft, they can open a new pull request, which must have "FINAL" in the pull request name, to move the rule from `_drafts` to `_rules`. This will publish the rule. The rule needs at least **3 approved votes** from other people than the editors before it can be merged. If new comments come in, the pull request must be closed without merging, and a new issue must be created to work on the fix.